### ISSUE TITLE
Crash to desktop during "Hah... You're not so big now"

### ISSUE THREAD

**Issue by Alderego** [dalaranwow/dalaran-wow#5158]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
The game crashed to the desktop whenever I used the quest item. This usually happened about 5-10 seconds after using it.

**Expected behaviour**: Tell us what should happen instead
The game should have just continued to run while adding 1 to the quest counter.

**Steps to reproduce the problem**:

1.  Use the quest item Crafty's Ultra-Advanced Proto-Typical Shortening Blaster in a Magmoth Crusher.
2. Enjoy the crash
3. Log back in and see that the counter went up 1 for the quest.

**Include proofs for this behaviour**
Reports attached below.

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
Quest: ID =11653 ,woheadlink: http://www.wowhead.com/quest=11653/hah-youre-not-so-big-now
Item: ID = 34812 ,woheadlink: http://www.wowhead.com/item=34812/craftys-ultra-advanced-proto-typical-shortening-blaster
Mob: ID = 25434 ,wowheadlink: http://www.wowhead.com/npc=25434/magmoth-crusher

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

[report1.txt](https://github.com/dalaranwow/dalaran-wow/files/1524791/report1.txt)
[report2.txt](https://github.com/dalaranwow/dalaran-wow/files/1524792/report2.txt)
[report3.txt](https://github.com/dalaranwow/dalaran-wow/files/1524793/report3.txt)
[report4.txt](https://github.com/dalaranwow/dalaran-wow/files/1524794/report4.txt)

**Comment by Lothloryen** [comment]
Hello there,

Thank you for your report, I can not reproduce such issue. It seems client-side related.

**Comment by TeridanDW** [comment]
#706
